chall02/pcuadrad.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import sys
+
+def check_string(txt):
+    for char in txt:
+        if not char.isalpha() and not char.isspace():
+            return 0
+    return 1
```

Comment:
You could have made this more pythonic with this oneliner:
```suggestion
    return txt.replace(" ", "").isalpha()
```
Remember that in python, you have booleans, you should use `True` and `False` instead of `1` and `0`